### PR TITLE
Symfony 2.7

### PR DESCRIPTION
**Pull request by chives**

### CODE REVIEW

**Comment by JarekW** [comment]
:+1:

**Comment by szymach** [comment]
:+1:

**Comment by chives** [comment]
@fsi-open

File: Behat/Context/DataContext.php [hunk 1]
```diff
@@ -44,7 +43,7 @@ public function createDatabase()
     }
 
     /**
-     * @AfterScenario
+     * aaa @AfterScenario
```
Comment (szymach) [inline Behat/Context/DataContext.php]: :)

**Comment by rn0** [comment]
:+1:

**Comment by JarekW** [comment]
:+1:

**Comment by szymach** [comment]
:+1:

**Pull request merged**